libbeat/_meta/config/setup.dsl.reference.yml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{header "Data Stream Lifecycle (DSL)"}}
+
+# Configure Data Stream Lifecycle to manage data streams while connected to Serverless elasticsearch. 
+# These settings are mutually exclusive with ILM settings which are not supported in Serverless projects.
+
+# Enable DSL support. Valid values are true, or false.
+#setup.dsl.enabled: true
+
+# Set the lifecycle policy name or pattern. For DSL, this name must match the data stream that the lifecycle is for.
+# The default data stream pattern is %{[beat.name]}-%{[beat.version]}"
```

Comment:
Were you intended to actually have `%{[beat.name]}-%{[beat.version]}` be templated or did you want this literal text in the docs?

I don't know that I've seen this format used elsewhere but I might be missing it.

Reply:
Ah, you're right, that's probably not the right way to do that...

Reply:
So, turns out there's no template variable for `.VersionName` or anything similar, so the next best thing is `metricbeat-%{[agent.version]}` an such.